scripts/sysman/power.yml

```
@@ -257,6 +257,7 @@ name: GetLimits
 details:
     - "The application may call this function from simultaneous threads."
     - "The implementation of this function should be lock-free."
+    - "Note: This function is being deprecated and replaced by $sPowerGetLimitsExt."
```

```suggestion
    - "Note: This function is deprecated and replaced by $sPowerGetLimitsExt."
```

I think it is more correct to say that it **is** deprecated, unless we really want to say that it it is **being** deprecated (at some point in the future, in other words not deprecated just yet)?

---

Agree with your suggestion @bashbaug . Changing.

---

One more thought: this should be guarded by version 1.5 also, correct?  Otherwise this will apply retroactively to all spec versions.

---

I was thinking on that. so something like

```cpp
- "1.5": "Note: This function is deprecated and replaced by $sPowerGetLimitsExt."
```

---

should we mark the previous two lines with 1.0, or just marking this one with 1.5 is enough?